src/styles/ButtonPanel.scss

```diff
@@ -0,0 +1,8 @@
+.ButtonPanel {
+  height: 100%;
+
+  .row-container {
```

Comment:
height is 100px ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/rowHeight.png))